src/xarm-commander.cpp

```diff
@@ -0,0 +1,157 @@
+#include <CLI11.hpp>
+#include <xarm/wrapper/xarm_api.h>
+
+#include <string>
+#include <iostream>
+
+int main(int argc, char **argv)
+{
+    XArmAPI *arm;
+    std::string port = "130.82.171.9";
+    int res = 1;
+
+    CLI::App app{"xarm-commander: a command line tool for controlling the xArm7."};
+
+    // print defaults on help
+    app.option_defaults()->always_capture_default();
+
+    // ===== FLAGS =====
+    bool print_mode = false;
+    app.add_flag("-v, --verbose", print_mode, "Verbose mode."); // TODO: multilevel verbose modes + logger
```

Comment:
> logger
Yes please, instead of `std:cout` 😄

Reply:
Will be done in branch: [feat/logging](https://github.com/Interactions-HSG/xarm-commander/tree/feat/logging)